src/systems/ec/implementations/react/packages/timeline/src/Timeline.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import VanillaTimeline from '@ecl/ec-component-timeline/ec-component-timeline';
+import Button from '@ecl/ec-react-component-button/Button';
+
+import TimelineItem from './TimelineItem';
+
+export default class Timeline extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+    this.timeline = null;
+    this.timelineRef = React.createRef();
+  }
+
+  componentDidMount() {
+    this.timeline = new VanillaTimeline(this.timelineRef.current);
+    this.timeline.init();
+  }
+
+  componentWillUnmount() {
+    if (this.timeline) this.timeline.destroy();
+  }
+
+  render() {
+    const { items, button, className, ...props } = this.props;
+
+    const classNames = classnames(className, 'ecl-timeline');
+
+    const showAllButton = (
+      <Fragment>
+        <br />
+        <Button
+          {...button}
+          className={classnames(button.className, 'ecl-timeline__button')}
+          data-ecl-timeline-button
+        />
+      </Fragment>
+    );
+
+    return (
+      <section
+        {...props}
+        className={classNames}
+        ref={this.timelineRef}
+        data-ecl-timeline
+      >
+        <ol className="ecl-timeline__list" data-ecl-timeline-list>
+          {items.map((item, index) => (
+            <TimelineItem
+              key={item.label}
+              label={item.label}
+              className={classnames({
+                [`ecl-timeline__item--collapsed`]: index > 2,
+              })}
+            >
+              {item.content}
+
+              {index === 2 && showAllButton}
```

Comment:
The button is not part of the content per se. I would keep it outside of the list or as a special list item instead of embedding it within the content of an existing item.